sunbird_service_api_test/src/test/java/org/sunbird/integration/test/page/section/ReadPageSectionTest.java

```diff
@@ -0,0 +1,65 @@
+package org.sunbird.integration.test.page.section;
+
+import com.consol.citrus.annotations.CitrusTest;
+import com.consol.citrus.testng.CitrusParameters;
+import org.springframework.http.HttpStatus;
+import org.sunbird.integration.test.common.BaseCitrusTestRunner;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+public class ReadPageSectionTest extends BaseCitrusTestRunner {
```

Comment:
Add testReadPageSettingFailureWithoutAccessToken in ReadPageSettingsTest. Similarly, create ReadAllPagesSettingsTest.